packages/commonwealth/client/scripts/views/components/NewThreadForm/NewThreadForm.tsx

```diff
@@ -339,17 +341,21 @@ export const NewThreadForm = () => {
                 />
               )}
 
-              <ReactQuillEditor
-                contentDelta={threadContentDelta}
-                setContentDelta={setThreadContentDelta}
-                isDisabled={isRestrictedMembership || !user.activeAccount}
-                tooltipLabel={
-                  typeof disabledActionsTooltipText === 'function'
-                    ? disabledActionsTooltipText?.('submit')
-                    : disabledActionsTooltipText
-                }
-                placeholder="Enter text or drag images and media here. Use the tab button to see your formatted post."
-              />
+              {!newEditor && (
+                <ReactQuillEditor
+                  contentDelta={threadContentDelta}
+                  setContentDelta={setThreadContentDelta}
+                  isDisabled={isRestrictedMembership || !user.activeAccount}
+                  tooltipLabel={
+                    typeof disabledActionsTooltipText === 'function'
+                      ? disabledActionsTooltipText?.('submit')
+                      : disabledActionsTooltipText
+                  }
+                  placeholder="Enter text or drag images and media here. Use the tab button to see your formatted post."
+                />
+              )}
+
+              {newEditor && <Editor mode="desktop" placeholder="" />}
```

Comment:
what happens on mobile mode?

Reply:
Nothing yet.  The integration will be done in v3.